draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -448,12 +456,94 @@ Steps:
 20.  y = u * x (mod p)       // ux
 21.  y = y + v (mod p)       // ux + v
 22. Output (x, y)
-
 ~~~
 
 ### Shallue-Woestijne-Ulas Method {#swu}
 
-((TODO: write this section))
+The Shallue-Woestijne-Ulas (SWU) method, originated in part by
+Shallue and Woestijne {{SW06}} and later simplified and extended by Ulas {{SWU07}},
+deterministically encodes an artbirary string to a point on a curve. 
+This algorithm works for any curve over F_{p^n}. Given curve equation 
```

Comment:
I guess we should clarify (for this, and for the others) that it only works for any Weierstrauss curve? Or maybe "curve in Weierstrauss form"? Although I expect you could lift the encoding by just applying the suitable transformation?

Reply:
Right -- the Weierstrauss requirement applies for all (?) the algorithms in the document. I think it's easier to keep that pattern, and then later -- in the TODO section -- describe how to transform from one form to another.

Reply:
Doh, scratch that. Elligator2 is given for Montgomery curves.

Reply:
@samscott89 Are the latest changes good to go?